### ISSUE TITLE
Missing accessibilitye

### ISSUE THREAD

**Issue by JuliusJuliussen** [googlecodelabs/feedback#1579]
Not only the program still missing some accessibility, what is the "-1button"?

Also this page is not the most beautiful when it comes to accessibility ...

Diagrams, Pictures, and a lot of them, but no explanation.

For now, I will let the program unused, I can't even find some shortcut-list.

Even more strange, you can't access commands from the menubar by pressing enter ...

You are forced to use the commands, and that is a problem, if you can't remember.

I am sure this will be fine, later, but so far, I will stick with vscode.

**Comment by meteatamel** [comment]
Which codelab are you refering to @JuliusJuliussen ?

**Comment by meteatamel** [comment]
Closing now. Please open again with more details.
